app/cdap/components/GridTable/components/TransformationToolbar/index.tsx

```diff
@@ -0,0 +1,132 @@
+/*
+ *  Copyright © 2022 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+import { IconButton, Typography, Tooltip, TextField } from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+import { default as React, useState } from 'react';
+import NestedMenu from 'components/GridTable/components/NestedMenu';
+import { ITransformationToolBarProps } from 'components/GridTable/components/TransformationToolbar/types';
+import {
+  Divider,
+  Expand,
+  LongDivider,
+} from 'components/GridTable/components/TransformationToolbar/iconStore';
+import { useStyles } from 'components/GridTable/components/TransformationToolbar/styles';
+import FunctionToggle from 'components/GridTable/components/FunctionNameToggle';
+import { nestedMenuOptions } from 'components/GridTable/components/TransformationToolbar/utils';
+import { IMenuItem } from 'components/GridTable/components/MenuItemComponent/types';
+
+export default function({
+  columnType,
+  submitMenuOption,
+  setShowBreadCrumb,
+  showBreadCrumb,
+}: ITransformationToolBarProps) {
+  const classes = useStyles();
+  const [isShowNames, setIsShowName] = useState<boolean>(false);
+  const [anchorEl, setAnchorEl] = useState<HTMLElement>(null);
+  const [selectedMenuOptions, setSelectedMenuOptions] = useState<IMenuItem[]>([]);
+
+  const handleMenuOpenClose = () => {
```

Comment:
can we rename this function to toggleMenu

Reply:
Done